charts/kube-master/templates/dex.yaml

```diff
@@ -0,0 +1,246 @@
+{{ if .Values.dex.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+  name: {{ include "master.fullname" . }}-dex 
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "master.fullname" . }}-dex 
+      release: {{ .Release.Name }}  
+  template:
+    metadata:
+      labels:
+        app: {{ include "master.fullname" . }}-dex 
+        release: {{ .Release.Name }} 
```

Comment:
Please add a checksum annotation here. In case the configmap changes this will then trigger a restart. See for example: https://github.com/sapcc/kubernikus/blob/master/charts/kube-master/templates/_helpers.tpl#L31-L38

Reply:
dex configmap is separated and checksum is added to the deployment: https://github.com/sapcc/kubernikus/pull/464/commits/ab997d3abb599ada9b7e84e652f4b8b80111e944#diff-d238b1eb95a4202a4d4d6b0ca52509dcR22